.devcontainer/Dockerfile

```diff
@@ -33,7 +34,7 @@ RUN \
 
 # GoLang
 RUN \
-    wget -c https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz -O - | tar -xz -C /usr/local \
+    wget -c https://dl.google.com/go/go1.18.1.linux-${TARGETARCH}.tar.gz -O - | tar -xz -C /usr/local \
```

Comment:
We didn't switch the codebase to Go 1.18 yet - we're still at 1.16 here, I've seen some build failures related to static checking tooling we use. 

Change to 1.18 should be a more global one - CI, releasing, go.mod

Reply:
0.15 didn't work so I upgraded to the latest one.

I'll try 0.16.